README.md

```diff
@@ -43,7 +43,7 @@ Our templates offer a rich boilerplate to jump start Go Gin-based application de
 
 ## Test
 
-Execute all unit tests:
+- Execute all unit tests. Note that the **main** branch is used. Refer to [this wiki page](https://github.com/nimblehq/gin-templates/wiki/Commands) for instructions on how to use a different branch.
```

Comment:
Same as previous comment

Reply:
Good catch. Fixed in 65a9b78c2557ed0af301851210e84459452c4c1b